File: Selenium/Activities/Activity1.py
from selenium import webdriver
from selenium.webdriver.common.by import By

# Launch Chrome browser
driver = webdriver.Chrome()

# Open the website
driver.get("https://training-support.net")

# Print title of the home page
print("Home Page Title:", driver.title)

# Click the About Us button
driver.find_element(By.LINK_TEXT, "About Us").click()

# Print title of the new page
print("About Us Page Title:", driver.title)

# Close the browser
driver.quit()

File: Selenium/Activities/Activity1.java
package FST_Coding_Training.FST_coding;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.chrome.ChromeDriver;

public class Activity1 {

    public static void main(String[] args) {

        WebDriver driver = new ChromeDriver();

        // Open the website
        driver.get("https://training-support.net");

        // Print home page title
        System.out.println("Home Page Title: " + driver.getTitle());

        // Click About Us button
        WebElement aboutUs = driver.findElement(By.linkText("About Us"));
        aboutUs.click();

        // Print new page title
        System.out.println("About Us Page Title: " + driver.getTitle());

        driver.quit();
    }
}